src/main/scala/com/ibm/couchdb/Model.scala

```diff
@@ -65,14 +63,12 @@ case class CouchDocsIncludesMissing[K, V, D](offset: Int,
 }
 
 case class CouchAttachment(content_type: String,
-                           revpos: Int,
-                           digest: String,
+                           revpos: Int = -1,
+                           digest: String = "",
                            data: String = "",
                            length: Int = -1,
                            stub: Boolean = false) {
-  private val decoder = new BASE64Decoder()
-
-  def toBytes: Array[Byte] = decoder.decodeBuffer(data)
+  def toBytes: Array[Byte] = java.util.Base64.getDecoder.decode(data)
```

Comment:
any reason for the full package name for Base64 here? should we just `import java.util.Base64`